CHANGES.rst

```diff
@@ -1691,8 +1691,8 @@ astropy.io.votable
 astropy.modeling
 ^^^^^^^^^^^^^^^^
 
-- Added a state attribute to models to allow preventing the synching of
-  constraint values from the constituent models. This synching can
+- Added a state attribute to models to allow preventing the syncing of
```

Comment:
So is this resolved or needs to be reverted?

Reply:
Probably easiest to just let it be. It would not be worth making an exception for this even if that was possible.

Indeed, this is really the main worry I have about using codespell, but I guess having the allowed misspelled word list will have to do.

Reply:
So good to keep ``syncing``? Resolve this comment?

Reply:
Sounds good!  I'll resolve this.  At the very least, this will improve consistency in how it's spelled, which is probably most important.